tests/test_loadDiaCatalogs.py

```diff
@@ -390,6 +395,25 @@ def testGetPixelRanges(self):
             self.assertEqual(normPix, testPix)
             self.assertEqual(flipPix, testPix)
 
+    def test_apdbSchema(self):
+        """Test that the default DiaSource schema from dax_apdb agrees with the
+        column names defined here in ap_association/data/DiaSource.yaml.
+        """
+        functorFile = _data_file_name("DiaSource.yaml", "ap_association")
+        apdbSchemaFile = _data_file_name("apdb-schema.yaml", "dax_apdb")
+        with open(apdbSchemaFile) as yaml_stream:
+            table_list = list(yaml.safe_load_all(yaml_stream))
+            for table in table_list:
+                if table['table'] == 'DiaSource':
+                    apdbSchemaColumns = [column['name'] for column in table['columns']]
+                    break
+        with open(functorFile) as yaml_stream:
+            diaSourceFunctor = yaml.safe_load_all(yaml_stream)
+            for functor in diaSourceFunctor:
+                diaSourceColumns = [column for column in list(functor['funcs'].keys())
+                                    if column not in self.ignoreColumns]
+                self.assertLess(set(diaSourceColumns), set(apdbSchemaColumns))
```

Comment:
This assert should go outside the `for` loop: you want to build up the list of column names from the `DiaSource.yaml` file, and then check against that whole list.